src/demos/demo1.ts

```diff
@@ -197,34 +208,61 @@ function main() {
       wheel
         .copy()
         .setMountPoint(RobotSpecs.WheelMountingPoint.LEFT_REAR)
-        .setMountOffset({ x: -0.0075, y: -0.25, z: -0.033 })
+        .setMountOffset({ x: -0.0075, y: -0.05, z: -0.033 })
     )
     .addWheel(
       "right-drive",
       wheel
         .copy()
         .setMountPoint(RobotSpecs.WheelMountingPoint.RIGHT_FRONT)
-        .setMountOffset({ x: 0.0075, y: -0.25, z: 0.033 })
+        .setMountOffset({ x: 0.0075, y: -0.05, z: 0.033 })
     )
     .addWheel(
       "right-drive",
       wheel
         .copy()
         .setMountPoint(RobotSpecs.WheelMountingPoint.RIGHT_REAR)
-        .setMountOffset({ x: 0.0075, y: -0.25, z: -0.033 })
+        .setMountOffset({ x: 0.0075, y: -0.05, z: -0.033 })
     )
     .addMotor("left-drive", motor)
     .addMotor("right-drive", motor.copy().setChannel(1));
 
   const spec = robotBuilder.generateSpec();
-  spec.customMesh = {
-    filePath: "assets/models/edubot-lores.gltf",
-    rotation: {
-      x: -Math.PI / 2,
-      y: 0,
-      z: Math.PI / 2,
+  // spec.customMesh = {
+  //   filePath: "assets/models/edubot-lores.gltf",
+  //   rotation: {
+  //     x: -Math.PI / 2,
+  //     y: 0,
+  //     z: Math.PI / 2,
+  //   },
+  // };
+
+  let gripperGrabChannel = 0;
+  let gripperHeldChannel = 1;
+
+  spec.mechanisms = [
```

Comment:
I suppose we want to wrap this into some sort of `MechanismBuilder`, for consistency?  But maybe it's best to land a first version of the gripper mechanism - so that we can integrate and test it inside the Simulator - and leave cosmetic changes for follow-up PRs?

Reply:
yeah that's a good plan.